setup.py

```diff
@@ -94,5 +100,5 @@ def text(*args, **kwargs):
         
         'zope.testing',
         ],
-    extras_require={'test':'zc.relationship >= 2.0c1'},
+    extras_require={'test':'zc.relationship'},
```

Comment:
I believe that the condition here still should remain (`>= 2.0c1`) because the versions below that will *not* work correctly as a test dependency, no matter how rare is the possibility that someone gets those old versions.
Correct metadata is important.